tasks/configure-swift-backups.yml

```diff
@@ -0,0 +1,36 @@
+- name: upload tarsnap key
+  template:
+    src: "tarsnap.key.j2"
+    dest: "{{ OPENCRAFT_BACKUP_SWIFT_TARSNAP_KEY_LOCATION }}"
+    owner: "www-data"
+    group: "root"
+
+- name: create backup folders
+  file:
+    name: "{{ item }}"
+    owner: "www-data"
+    group: "root"
+    state: "directory"
+  with_items:
+    - "{{ OPENCRAFT_BACKUP_SWIFT_TARGET }}"
+    - "{{ OPENCRAFT_BACKUP_SWIFT_TARSNAP_CACHE_LOCATION }}"
+
+# NOTE: These two tasks are optional.
+
+- name: mount volume
+  mount:
+    fstype: "ext4"
+    state: "mounted"
+    name: "{{ OPENCRAFT_BACKUP_SWIFT_TARGET }}"
+    src:  "{{ OPENCRAFT_BACKUP_SWIFT_MOUNT_DEVICE }}"
+  when: "{{ OPENCRAFT_BACKUP_SWIFT_MOUNT_DEVICE|length }}"
```

Comment:
This will fail when `OPENCRAFT_BACKUP_SWIFT_MOUNT_DEVICE: null`:

> "The conditional check '{{ OPENCRAFT_BACKUP_SWIFT_MOUNT_DEVICE|length }}' failed. The error was: an unexpected type error occurred. Error was object of type 'NoneType' has no len().

It's ok to just say:

` when: "{{ OPENCRAFT_BACKUP_SWIFT_MOUNT_DEVICE }}"`